contracts/identityManager/AlastriaIdentityManager.sol

```diff
@@ -49,9 +55,9 @@ contract AlastriaIdentityManager is AlastriaIdentityServiceProvider, AlastriaIde
         alastriaPublicKeyRegistry = AlastriaPublicKeyRegistry(_publicKeyRegistry);
         AlastriaProxy identity = new AlastriaProxy();
         identityKeys[_firstIdentityWallet] = address(identity);
+        firstIdentityWallet = _firstIdentityWallet;
```

Comment:
```suggestion
        firstIdentityWallet = identity;
```

It should be the proxy address, isn't it?

Reply:
I think this should be address(identity)